src/main/resources/com/chikli/hudson/plugin/naginator/NaginatorPublisher/config.jelly

```diff
@@ -32,8 +32,7 @@
             </f:entry>
             <j:if test="${descriptor.isMatrixProject(it)}">
                 <f:entry title="${%How to apply the regular expression to matrix}" field="regexpForMatrixStrategy">
-                    <!-- unfortunatelly, f:enum doesn't support doCheckXxxx -->
-                    <f:select />
+                    <f:select default="TestParent"/>
```

Comment:
context: https://github.com/jenkinsci/naginator-plugin/blob/naginator-1.18.2/src/main/java/com/chikli/hudson/plugin/naginator/NaginatorPublisher.java#L75

Reply:
Without this when adding the Publisher for the first time an exception is thrown and the UI breaks.